package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "private": true,
+  "scripts": {
+    "postinstall": "elm-tooling install"
+  },
+  "devDependencies": {
+    "elm-review": "^2.7.6",
```

Comment:
It still pains me so much that for one tool, we need to bring the whole world via npm. Without the need to go through npm to install elm-review, we could have gotten rid of all of it ^^. One day we'll get rid of npm ...

Reply:
One day we'll rewrite it in roc ;)

Reply:
Actually I had investigated a possible way of extending elm-tooling with js projects using something like vercel ncc. But that's for another time, if I ever get time to work on that with Simon.